Lombiq.Tests.UI/Models/VisualVerificationMatchApprovedContext.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace Lombiq.Tests.UI.Models;
 
-public class VisualVerificationMatchApprovedContext
+public partial class VisualVerificationMatchApprovedContext
```

Comment:
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/cs0260?f1url=%3FappId%3Droslyn%26k%3Dk(CS0260)

Reply:
![image](https://github.com/Lombiq/UI-Testing-Toolbox/assets/92299130/8d1692e2-0e8b-4f37-a1ef-bb3aea51d8c3)

Reply:
Ah, that's for `GeneratedRegex`, because it generates a second class. OK then.